sidekiq-limit_fetch.gemspec

```diff
@@ -6,20 +9,24 @@ Gem::Specification.new do |gem|
   gem.email         = 'dean@deanpcmad.com'
   gem.summary       = 'Sidekiq strategy to support queue limits'
   gem.homepage      = 'https://github.com/deanpcmad/sidekiq-limit_fetch'
-  gem.description   = "Sidekiq strategy to restrict number of workers which are able to run specified queues simultaneously."
+  gem.description   = 'Sidekiq strategy to restrict number of workers which are able to run specified ' \
+                      'queues simultaneously.'
 
-  gem.metadata["homepage_uri"] = gem.homepage
-  gem.metadata["source_code_uri"] = "https://github.com/deanpcmad/sidekiq-limit_fetch"
-  gem.metadata["changelog_uri"] = "https://github.com/deanpcmad/sidekiq-limit_fetch/blob/master/CHANGELOG.md"
+  gem.metadata['homepage_uri'] = gem.homepage
+  gem.metadata['source_code_uri'] = 'https://github.com/deanpcmad/sidekiq-limit_fetch'
+  gem.metadata['changelog_uri'] = 'https://github.com/deanpcmad/sidekiq-limit_fetch/blob/master/CHANGELOG.md'
 
-  gem.files         = `git ls-files`.split($/)
-  gem.test_files    = gem.files.grep %r{^spec/}
-  gem.require_paths = %w(lib)
+  gem.files         = `git ls-files`.split($INPUT_RECORD_SEPARATOR)
+  gem.require_paths = %w[lib]
+
+  gem.required_ruby_version = '>= 2.7.0'
```

Comment:
To respect the `Gemspec/RequiredRubyVersion` cop, I added this, according to:

https://github.com/deanpcmad/sidekiq-limit_fetch/blob/9bf29b248458c80593939f937c67c42ad4fae7e1/.github/workflows/ci.yml#L17